src/ImageSharp/Formats/Gif/LzwDecoder.cs

```diff
@@ -254,5 +263,8 @@ public void Dispose()
             this.suffix.Dispose();
             this.pixelStack.Dispose();
         }
+
+        [MethodImpl(InliningOptions.ColdPath)]
+        public static void ThrowBadMinimumCode() => throw new InvalidImageContentException("Gif Image does not contain a valid LZW minimum code.");
```

Comment:
Maybe this should be in `GifThrowHelper`?

Reply:
Yeah, forgot about that class will move